INDEX

```diff
@@ -1,4 +1,14 @@
-{
+{	
+        "adguardhome": {
```

Comment:
Can you please fix the indentation so that it matches with rest of the file ? ( tabs/spaces )